### PR TITLE
[UI] Bank Transfer page implementation 

### PR DESCRIPTION
**Pull request by asapconet**
### Main 
- Created designs fitting all screens **[768 & 1024]**
### New 
- BankPayouts component & transfer route[folder]
### Bug
- Extra white space from the Top and RHS of the screen.

**Desktop**
image

**Mobile**
image

### CODE REVIEW

File: components/GetstartedCard/index.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const PlainCard = ({ desc, title, route }: MyProps) => {
 // the difference the 1st function with the rest is it has an icon on the top
 
 // 1 Start integrating
-export default function GetStarted(): JSX.Element {
+export default function GetStarted(props: any): JSX.Element {
   return (
     <section
       className={classNames(
```
Comment (MeisterJustice) [inline components/GetstartedCard/index.tsx:69]: Is there a reason for the props? Don't see where it's being used

File: components/Sidebar/SidebarMobile.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+const SidebarMobile = (): JSX.Element => {
+  const router = useRouter()
+
+  return (
+    <div className='flex flex-col items-start justify-between space-x-10'>
+      <ul className='h-[34rem] space-y-4 mb-2'>
+        {sidebar.map(({ label, id, icon, children }: SidebarItem) => {
+          const splitPath = router.asPath.split('/')
+          const isActive = splitPath[2] === id
+          // console.log(id)
```
Comment (MeisterJustice) [inline components/Sidebar/SidebarMobile.tsx:18]: please remove all console logs

File: components/BankPayouts/index.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        </label>
+        <input
+          type={type}
+          id={id}
+          className={classNames(
+            Styles.input,
+            'placeholder:px-5  placeholder:font-proxima',
+          )}
+          placeholder='Enter your email address'
+        />
```
Comment (MeisterJustice) [inline components/BankPayouts/index.tsx:35]: I don't see any `onChange func`. I'm also wondering if it'll be better to have this label and input as a reusable component? Something like `LzInput`

**Review by MeisterJustice — CHANGES_REQUESTED** [review]

File: components/GetstartedCard/index.tsx (hunk 1 above)
Comment (asapconet) [reply components/GetstartedCard/index.tsx:69]: Ooo no worries I'll take 'em off.

File: components/BankPayouts/index.module.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.input {
+    width: 412px;
+    height: 48px;
+
+    background: #FFFFFF;
+    /* Wireframe/Light Grey 2 */
+
+    border: 1px solid #DFDFDF;
```
Comment (Frontend-io) [inline components/BankPayouts/index.module.scss:8]: Please use the theme() function from tailwind to add colors.  Please check this out: https://tailwindcss.com/docs/using-with-preprocessors

File: components/BankPayouts/index.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+export const PayoutLHS = ({ title, desc, label, id, type }: MyProps) => {
+  return (
+    <>
+      <h1 className='heading-3-b lg:heading-2-b'>{title}</h1>
+      <p className='paragraph-2'>{desc}</p>
+      <form className='flex flex-col space-y-6'>
+        <LzInput type={type} label={label} id={id} />
+        <LzButton
+          variant='pri'
+          rightIcon={<FaAngleRight />}
```
Comment (Frontend-io) [inline components/BankPayouts/index.tsx:26]: You don't need to pass a value to the `rightIcon`. There's already an icon in the component. pass the prop without a value. Please look at the button component to see how it works

File: components/BankPayouts/data.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    title: string
+    desc: string
+}
+
+export const periphery: MyProps[] = [{
+    id: 'email',
+    type: 'email',
+    label: 'email',
+    title: 'Get Notified When Bank Payout is Live',
+    desc: 'Bank Payout to multiple countries in USD and other local currencies is coming soon. Reach out to developers@lazerpay.finance if you have any questions regarding this'
```
Comment (Frontend-io) [inline components/BankPayouts/data.ts:14]: To add formatting to the link, please use html code. Then render the value on the text as 
```js
{
   desc: `Bank Payout to multiple countries in USD and other local currencies is coming soon. Reach out to <a href="mailto:developers@lazerpay.finance">developers@lazerpay.finance</a> if you have any questions regarding this`
}
```
```jsx
<p className='paragraph-2' dangerouslySetInnerHTML={{__html: desc}} />
```

File: components/BankPayouts/index.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+interface MyProps {
+  title: string
+  desc: string
+  label: string
+  type: string
+  id: string
+}
+
+export const PayoutLHS = ({ title, desc, label, id, type }: MyProps) => {
```
Comment (Frontend-io) [inline components/BankPayouts/index.tsx:17]: If there wont be any special logic in this component, just return the component directly. Eg

```jsx
const MyComponent = ()=> (
<div>Content</div>
)
```

File: components/BankPayouts/index.tsx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+          size='lg'
+        >
+          Notify me
+        </LzButton>
+      </form>
+    </>
+  )
+}
+
+const BankPayouts = () => {
```
Comment (Frontend-io) [inline components/BankPayouts/index.tsx:37]: Same here with returning component

**Review by Frontend-io — CHANGES_REQUESTED** [review]
Weldone man. Please see the comments attached. And lastly, please import the image and use as svg, not png

File: components/UI/input.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+interface MyProps {
+    id?: string
+    type?: string
+    label: string
+    placeholder?: string
+    onChange?: () => void
+}
+
+const LzInput = ({ label, id, type, placeholder, onChange }: MyProps) => {
```
[the remark this answers is not in the document]
Comment (asapconet) [reply components/UI/input.tsx:12]: Seen, thanks!

File: components/BankPayouts/index.tsx (hunk 7 above)
Comment (asapconet) [reply components/BankPayouts/index.tsx:17]: Sorted!

File: components/BankPayouts/index.tsx (hunk 8 above)
Comment (asapconet) [reply components/BankPayouts/index.tsx:37]: Sorted!

File: components/BankPayouts/index.module.scss (hunk 4 above)
Comment (asapconet) [reply components/BankPayouts/index.module.scss:8]: my bad, totally didn't remember.
sorted now!

**Comment by asapconet** [comment]
> 

Thanks boss, all observations reviewed and sorted!

**Review by Frontend-io — COMMENTED** [review]
Looking good! Just one more tiny but left, the validation for the email input. Let me know if you can add that, otherwise i'll merge and add myself. 

We are currently using `react-hook-form` and yup for form validation. Please see here for an example: https://codesandbox.io/s/wbybv

Official Website:  https://react-hook-form.com/

**Comment by asapconet** [comment]
> 

I’ll be on it right away boss

**Comment by Frontend-io** [comment]
Looks good!

**Pull request merged**